app/controllers/rentals_controller.rb

```diff
@@ -29,26 +31,7 @@ def show
 
   private
 
-  def most_recent_rental
-    if session[:cart]
-      find_items
-    else
-      {}
-    end
-  end
-
-  def find_items
-    item_ids = session[:cart].keys.map { |key| key.to_i }
-    @items = Item.where(id: item_ids)
-  end
-
-  def calculate_total_price(items)
-    item_prices = items.map { |item| item.price * session[:cart].fetch(item.id.to_s) }
-    price_per_day = item_prices.reduce(:+)
-    @total_price = (price_per_day * session[:cart]['days'])
-  end
-
   def rental_params
-    {user_id: session[:user_id], days_rented: session[:cart]['days'], total_price: @total_price}
+    {user_id: session[:user_id], days_rented: session[:cart]['days'], total_price: @cart.total_price, status: 'order placed'}
```

Comment:
Line is too long. [125/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.